python/tk_multi_workfiles/step_list_filter.py

```diff
@@ -224,21 +223,26 @@ def _ensure_widgets_for_entity_type(self, entity_type):
             # Turn it on if it was in the step saved filters
             # We do this before the toggled signal is connected to not emit
             # un-wanted signals.
-            if step["id"] in self._saved_filter_step_ids:
+            if step["id"] in self._current_filter_step_ids:
                 widget.setChecked(True)
-            widget.toggled.connect(self._on_step_filter_toggled)
+            widget.toggled.connect(lambda value, step_id=step["id"] : self._on_step_filter_toggled(step_id, value))
             item = QtGui.QListWidgetItem("", self._list_widget)
             item.setData(QtCore.Qt.UserRole, step)
             self._list_widget.setItemWidget(item, widget)
             self._step_widgets[entity_type].append(widget)
         return self._step_widgets[entity_type]
 
-    def _on_step_filter_toggled(self, value=None):
+    def _on_step_filter_toggled(self, step_id, value=None):
```

Comment:
`value` seems like an odd name for a boolean parameter - could we rename it to something clearer?